podcasts/PlayBufferManager.swift

```diff
@@ -17,10 +17,10 @@ class PlayBufferManager {
         let itemThatWouldHavePlayedNext = playBuffer.pop()
         removeAll()
         // if possible, leave the next item that would have played on the buffer, so we can fade it out
-        if var item = itemThatWouldHavePlayedNext {
-            item.shouldFadeOut = true
-            playBuffer.push(item)
-        }
+//        if var item = itemThatWouldHavePlayedNext {
```

Comment:
I believe the fade-out is what's causing the issue.

After commenting those lines, I'm not able to reproduce the issue anymore.

@danielebogo could you please give it a try?